cli/command/container/list.go

```diff
@@ -59,21 +59,12 @@ func newListCommand(dockerCli *command.DockerCli) *cobra.Command {
 	return &cmd
 }
 
-type preProcessor struct {
-	types.Container
```

Comment:
Personally `map[string]bool` is hard to understand...

How about just avoid embedding `types.Container`?
i.e.
```go
type preProcessor struct {
    c types.Container
    opts *types.ContainerListOptions
}
```
```